client.go

```diff
@@ -443,6 +489,76 @@ func (client *Client) CfInfo(key string) (map[string]int64, error) {
 	return ParseInfoReply(redis.Values(conn.Do("CF.INFO", key)))
 }
 
+// TdCreate - Allocate the memory and initialize the t-digest
+func (client *Client) TdCreate(key string, compression int64) (string, error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
+	return redis.String(conn.Do("TDIGEST.CREATE", key, compression))
+}
+
+// TdReset - Reset the sketch to zero - empty out the sketch and re-initialize it
+func (client *Client) TdReset(key string) (string, error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
+	return redis.String(conn.Do("TDIGEST.RESET", key))
+}
+
+// TdAdd - Adds one or more samples to a sketch
+func (client *Client) TdAdd(key string, samples map[string]float64) (string, error) {
+	conn := client.Pool.Get()
+	defer conn.Close()
+	args := redis.Args{key}
+	for k, v := range samples {
+		args = args.Add(k, v)
+	}
+	reply, err := conn.Do("TDIGEST.ADD", args...)
+	return redis.String(reply, err)
+}
+
+// TdMerge - Merges all of the values from 'from' to 'this' sketch
+func (client *Client) TdMerge(toKey string, fromKey string) (string, error) {
```

Comment:
Suggest to change signature from `TdMerge(toKey string, fromKey string) (string, error)` to `TdMerge(toKey string, fromKey string) (bool, error)`
Given that the commands replies with:

```
 OK on success, error otherwise 
 ```
 https://oss.redislabs.com/redisbloom/master/TDigest_Commands/#tdigestmerge